src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calledCount = 0;
```

Comment:
this variable can be called just `count` in this case, it is pretty clear from your code what it counts